definitions/npm/underscore.string_v3.x.x/flow_all/underscore.string_v3.x.x.js

```diff
@@ -0,0 +1,87 @@
+declare module "underscore.string" {
+
+  declare type $npm$wrap$options = {
+    width?: number,
+    seperator?: string,
+    trailingSpaces?: boolean,
+    cut?: boolean,
+    preserveSpaces?: boolean,
+  };
+  declare type $npm$pad$type = 'left' | 'right' | 'both';
+
+  declare class S {
+    // TODO: fix it
+    static chain(string: string): any;
```

Comment:
I'm looking at the API docs, and I don't see that `chain()` is actually a static method?

It looks like instead the `exports` object is a function with lots of static methods on it? When the function is called, it returns an object with a version of those same methods (except they have a slightly different signature:

`toBoolean: (str: string) => boolean` vs `chain.toBoolean: () => boolean`
(note that one has an input param and the other doesn't)

In this case, you're probably better off defining both a class for `chain` objects, and a "callable object" for the `exports` object:

```js
declare module "underscore.string" {
  ...

  declare class Chain {
    toLowerCase(str: string): string;
    ...
  }

  declare module.exports: {
  };
}
```